tools/toolchains/__init__.py

```diff
@@ -344,7 +344,7 @@ def get_symbols(self):
 
             # Config support
             if self.has_config:
-                self.symbols.append('HAVE_MBED_CONFIG_H')
+                self.symbols.append('HAVE_MBED_CONFIG')
```

Comment:
This was left so the config system can raise the flag. The user app can use the macro to "know" that configuration is in place.